docs/deploying/rhoai.md

```diff
@@ -0,0 +1,105 @@
+# Using Red Hat OpenShift AI
+
+[Red Hat OpenShift AI](https://www.redhat.com/en/products/ai/openshift-ai) is a cloud-native AI platform that bundles together many popular model management projects, including [kserve](https://kserve.github.io/website/latest/).
+
+This example shows how to use kserve with RHOAI to deploy a model on OpenShift, using a modelcar image to load the model without requiring any connection to huggingface hub.
+
+## Deploying with kserve
+
+!!! note
+    **Prerequisite**: Ensure that you have a running Kubernetes cluster with RHOAI installed, image pull credentials for `registry.redhat.io/rhelai1`, and Spyre accelerators.
```

Comment:
This is nicer as a list (if formatting in the note allows 🤔 ):

```suggestion
    **Prerequisites**: 
    - A running Kubernetes cluster with RHOAI installed
    - Image pull credentials for `registry.redhat.io/rhelai1`
    - Spyre accelerators available in the cluster
```

Reply:
Shoot, doesn't seem to work like that at least

Reply:
<img width="779" alt="image" src="https://github.com/user-attachments/assets/6b76f3ad-7351-4dbe-8bc4-8d4cf014bbdc" />

I updated the suggestion, looks like it works with some indenting

```
!!! note

    **Prerequisites**:
      
      - A running Kubernetes cluster with RHOAI installed
      - Image pull credentials for `registry.redhat.io/rhelai1`
      - Spyre accelerators available in the cluster
```

Reply:
Nice, I also tried updating the title to `Prerequisites` instead of `Note`, wdyt?